src/javascript/lib/modernizr-inputtype.js

```diff
@@ -0,0 +1,3 @@
+/*! modernizr 3.1.0 (Custom Build) | MIT *
```

Comment:
This file is for us to check if a particular html5 input type is supported or not